_data/tagging-status.yml

```diff
@@ -3383,11 +3383,11 @@
 
  - name: siunitx
    type: package
-   status: unknown
-   issues:
-   tests: false
-   tasks: needs tests
-   updated: 2024-07-15
+   status: partially-compatible
+   issues: "does not complain, but produces separately formulas for numbers and units"
+   tests: true
+   tasks: needs even more tests
```

Comment:
should get a link to the issue opened

Reply:
note the top of the file has the expected type of each field:  `issues:` is a list of integers (the supplied text should be in a `comment:` field)

Reply:
link to #197 added while resolving merge conflict